Symfony/CS/ConfigurationException/InvalidFixerConfigurationException.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\ConfigurationException;
+
+use Symfony\CS\Console\Command\FixCommand;
+
+/**
+ * Exception thrown by Fixers on misconfiguration.
+ *
+ * @author SpacePossum
+ *
+ * @internal
+ */
+class InvalidFixerConfigurationException extends InvalidConfigurationException
+{
+    /**
+     * @param string $fixerName
+     * @param string $message
+     */
+    public function __construct($fixerName, $message)
+    {
+        parent::__construct(sprintf('[%s] %s', $fixerName, $message), FixCommand::EXIT_STATUS_FLAG_HAS_INVALID_FIXER_CONFIG);
```

Comment:
@SpacePossum here you pass two arguments, but parent constructor (`InvalidConfigurationException`) has only one parameter..